go/pkg/amqp/url_test.go

```diff
@@ -32,8 +32,9 @@ func ExampleParseURL() {
 		"amqps://host",
 		"/path",
 		"",
-		":1234",
-                // Taken out because the go 1.4 URL parser isn't the same as later
+		// Taken out because url.Parse error formatting changed in 1.13
```

Comment:
The "taken out" lines can just be deleted - they're not valid URLs, those forms were only ever supported for backwards compat and I'm pretty sure aren't needed anymore.

Reply:
Great, I'll remove those.